gallery.html

```diff
@@ -1,11 +1,57 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <link rel="stylesheet" type="text/css" href="gallery.css">
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Gallery</title>
 </head>
 <body>
-    
+    <nav>
+        <ul class="navbar">
+            <li><a href="index.html">Bio</a></li>
+            <li><a href="gallery.html">Gallery</a></li>
+            <li><a href="contact.html">Contact</a></li>
+        </ul>
+    </nav>
+
+    <div class="heading">
+        <h1>My Collection of Harry Potter Wands</h1>
+    </div>
+
+    <div class="images">
+        <div class="image">
+            <img src="images/harry_wand.jpg" alt="Harry Potter's Wand" height=300px>
+            <figcaption>Harry Potter's Wand</figcaption>
+        </div>
```

Comment:
you may consider using a `figure` tag here, which is more semantic than a div, and goes well with the `figcaption` tag